gradle-client/src/jvmMain/kotlin/org/gradle/client/ui/connected/actions/DeclarativeDomHelpers.kt

```diff
@@ -5,7 +5,7 @@ import org.gradle.internal.declarativedsl.dom.DeclarativeDocument.DocumentNode.E
 import org.gradle.internal.declarativedsl.dom.DeclarativeDocument.DocumentNode.PropertyNode
 
 val DeclarativeDocument.singleSoftwareTypeNode: ElementNode
-    get() = content.single() as ElementNode
+    get() = content.filterIsInstance<ElementNode>().single()
```

Comment:
It will still crash with empty content or more than one node, is that OK?

Reply:
Good point, Should be fixed now, PTAL